### ISSUE TITLE
Crash on triggering services

### ISSUE THREAD

**Issue by coolo** [openSUSE/open-build-service#7574]
```

/PROJECT_ROOT/app/models/user.rb:229→ session!
--
/PROJECT_ROOT/app/models/package.rb:795→ write_to_backend
/GEM_ROOT/gems/activesupport-5.2.3/lib/active_support/callbacks.rb:426→ block in make_lambda
/GEM_ROOT/gems/activesupport-5.2.3/lib/active_support/callbacks.rb:236→ block in halting_and_conditional
/GEM_ROOT/gems/activesupport-5.2.3/lib/active_support/callbacks.rb:517→ block in invoke_after
/GEM_ROOT/gems/activesupport-5.2.3/lib/active_support/callbacks.rb:517→ each
/GEM_ROOT/gems/activesupport-5.2.3/lib/active_support/callbacks.rb:517→ invoke_after
/GEM_ROOT/gems/activesupport-5.2.3/lib/active_support/callbacks.rb:133→ run_callbacks
/GEM_ROOT/gems/airbrake-8.0.1/lib/airbrake/rails/active_record.rb:20→ run_callbacks
/GEM_ROOT/gems/activesupport-5.2.3/lib/active_support/callbacks.rb:816→ _run_save_callbacks
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/callbacks.rb:342→ create_or_update
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/persistence.rb:275→ save
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/validations.rb:46→ save
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/transactions.rb:310→ block (2 levels) in save
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/transactions.rb:387→ block in with_transaction_returning_status
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/connection_adapters/abstract/database_statements.rb:265→ transaction
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/transactions.rb:212→ transaction
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/transactions.rb:385→ with_transaction_returning_status
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/transactions.rb:310→ block in save
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/transactions.rb:325→ rollback_active_record_state!
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/transactions.rb:309→ save
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/suppressor.rb:44→ save
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/persistence.rb:428→ block in update
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/transactions.rb:387→ block in with_transaction_returning_status
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/connection_adapters/abstract/database_statements.rb:267→ block in transaction
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/connection_adapters/abstract/transaction.rb:239→ block in within_new_transaction
/usr/lib64/ruby/2.5.0/monitor.rb:226→ mon_synchronize
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/connection_adapters/abstract/transaction.rb:236→ within_new_transaction
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/connection_adapters/abstract/database_statements.rb:267→ transaction
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/transactions.rb:212→ transaction
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/transactions.rb:385→ with_transaction_returning_status
/GEM_ROOT/gems/activerecord-5.2.3/lib/active_record/persistence.rb:426→ update
/PROJECT_ROOT/app/models/package.rb:464→ sources_changed
/PROJECT_ROOT/app/controllers/trigger_controller.rb:51→ runservice

**Comment by hennevogel** [comment]
Please do not self-assign priority labels, talk to you PO.

**Comment by hennevogel** [comment]
This is addressed in #6965

**Comment by vpereira** [comment]
merged
